scrapy/core/downloader/handlers/negotiation.py

```diff
@@ -37,11 +36,12 @@ def handshakeCompleted(self) -> None:
         self._negotiated_protocol_deferred.callback(self.transport)
 
 
+@implementer(IProtocolNegotiationFactory)
 class NegotiateProtocolFactory(Factory):
     def __init__(
-        self,
-        acceptable_protocols: List[bytes],
-        negotiated_protocol_deferred: Deferred
+            self,
+            acceptable_protocols: List[bytes],
+            negotiated_protocol_deferred: Deferred
```

Comment:
:lipstick: Why the additional indentation level? I think it was fine before. And I see you’ve made similar changes below. Was it an IDE suggestion or something?